__tests__/utils.test.js

```diff
@@ -160,16 +161,20 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/4")
       .expect(200)
       .then((response) => {
-        const data = response.body.articleData;
+        const data = response.body.article;
```

Comment:
Although this works the test is "brittle". If in the future you add a property to an article this test will fail because it will no longer be completely "equal". As you were testing previously was good 👍